tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/defaults/main.yml

```diff
@@ -7,4 +7,5 @@ windows_ami_name: Windows_Server-2019-English-Full-Base-*
 # see:
 # - https://github.com/mattclay/aws-terminator/pull/181
 # - https://github.com/ansible-collections/community.aws/pull/763
-s3_bucket_name: ssm-encrypted-test-bucket
+s3_bucket_name: "{{ tiny_prefix }}-ssm-encrypted-test-bucket"
```

Comment:
@gillg Thank you for working on this. `ssm-encrypted-test-bucket` has been added in this PR https://github.com/ansible-collections/community.aws/pull/763#issuecomment-971775249 since It is an encrypted bucket requiring up to ~24hour to be created (we have to use a permanent one). At the moment, the integration tests of the aws_ssm connection plugin have been also disabled because of other issues.
@jillr can you confirm please?

Reply:
24h to create a bucket 😨 !? I'm surprised but why not.
Don't you think it could be more revelent to create an unencrypted bucket by default, but add a resource policy to ensure "PUT" commands forces encryption ? Because if you enable encryption in bucket properties that means you object will be encrypted by default event if you don't ask for it. But it seems to don't be the use case of SSM plugin where we ask explicitely to encrypt new objects.

If there is no workaround what would be the next steps ? Do we assume to skip integration tests for this plugin for now ?
Just in case... I tested it localy in my context 😅 

In the meantime I try to work on other PRs because I use this plugin for the first time and I prefer make it stable and close all pending PRs since almost 1yr.